Card/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        minSdkVersion 21
```

Comment:
We had an issue with the PayPal Auth SDK with minSDKversions of less than 23. We can always update this before our 1.0 release, but probably worth chatting as a team about it.

Reply:
Ok sounds good. We currently support API 21 in the Braintree SDK and I think in previous versions we went all the way back to API 16. I did also hear of some merchants with heavy traffic on some of the older versions of both iOS and Android. We should definitely support what we can, but I agree this will be a trade-off to allow us to ship the best SDK possible.

Reply:
Actually, now that I'm thinking about it, we'll probably want to keep support for API level 21. The functions that invoke the PayPal paysheet require API 23 and up, so If we end up pulling in the PayPal Checkout SDK for the PayPal module, the merchant would have to have some fallback for API 21 and 22 if their app supported it.